verilog/analysis/symbol_table.cc

```diff
@@ -1008,18 +1007,20 @@ class SymbolTable::Builder : public TreeContextVisitor {
   SymbolTableNode* EmplaceElementInCurrentScope(const verible::Symbol& element,
                                                 absl::string_view name,
                                                 SymbolMetaType metatype) {
-    const auto p = current_scope_->TryEmplace(
+    const auto [kv, did_emplace] = current_scope_->TryEmplace(
         name, SymbolInfo{metatype, source_, &element});
-    if (!p.second && p.first->second.Value().is_port_identifier) {
-      p.first->second.Value().supplement_definitions.push_back(name);
-    } else if (!p.second) {
-      DiagnoseSymbolAlreadyExists(name, p.first->second);
+    if (!did_emplace) {
+      if (kv->second.Value().is_port_identifier) {
+        kv->second.Value().supplement_definitions.push_back(name);
+      } else {
+        DiagnoseSymbolAlreadyExists(name, kv->second);
+      }
     }
-    return &p.first->second;  // scope of the new (or pre-existing symbol)
+    return &kv->second;  // scope of the new (or pre-existing symbol)
   }
 
   // checks whether a given tag belongs to one of the listed tags
-  bool IsTagMatching(int tag, const std::vector<int>& tags) {
+  bool IsTagMatching(int tag, const std::initializer_list<int>& tags) {
```

Comment:
```suggestion
  bool IsTagMatching(int tag, std::initializer_list<int> tags) {
```
`std::initializer_list` is cheap to copy (it is implemented as 2 iterators/pointers or something similar).